src/lambda-edge/parse-auth/index.ts

```diff
@@ -50,7 +50,7 @@ export const handler: CloudFrontRequestHandler = async (event) => {
     } catch (err) {
         return {
             body: createErrorHtml('Bad Request', err.toString(), redirectedFromUri),
-            status: '400', // Note: do not send 403 (!) as we have CloudFront send back index.html for 403's to enable SPA-routing
+            status: '400',
```

Comment:
Should this be `403` now?

Reply:
It doesn't matter for our purpose.

Maybe a matter of taste but I think 400 is clear as the request is indeed "bad".

The regular 403 redirects to Cognito. 400 then means here: we're not even gonna redirect you, your request is messed up.

Reply:
I deployed the new version using SAM and tested it, all good. Also deployed through serverless application repository, all good.

I'm gonna merge this PR now. THANK YOU for reporting the issue.

Reply:
Thank you for your quick fixes! :+1: :smile: